snow/networking/timeout/manager.go

```diff
@@ -71,27 +71,33 @@ type Manager interface {
 func NewManager(
 	timeoutConfig *timer.AdaptiveTimeoutConfig,
 	benchlistMgr benchlist.Manager,
-	metricsNamespace string,
-	metricsRegister prometheus.Registerer,
+	requestReg prometheus.Registerer,
+	responseReg prometheus.Registerer,
```

Comment:
Should we try to merge these namespaces?

Reply:
I mainly just found it weird that we now have to pass in multiple registerers... The implications could be either changing some metrics or changing where metrics are specified... I didn't make the change here because I wasn't very happy with the alternatives